test/user.js

```diff
@@ -0,0 +1,134 @@
+const chai = require('chai'),
+  dictum = require('dictum.js'),
+  app = require('./../app'),
+  should = chai.should(),
+  expect = chai.expect,
+  User = require('./../app/models').User,
+  user = { name: 'name', lastname: 'lastname', email: 'test@wolox.co', password: '12345678' };
+
+describe('/users POST', () => {
+  it('should create a valid user and return 201', done => {
+    chai
+      .request(app)
+      .post('/users')
+      .send(user)
+      .end(function(err, res) {
+        expect(res).to.have.status(201);
+        res.should.be.json;
+        res.body.should.be.a('object');
+        res.body.user.should.have.property('name');
+        res.body.user.should.have.property('lastname');
+        res.body.user.should.have.property('id');
+        res.body.user.name.should.equal('name');
+        res.body.user.lastname.should.equal('lastname');
```

Comment:
also validate in the DB wether the user was really created or not, just import the model a query it